subo/main.go

```diff
@@ -2,17 +2,11 @@ package main
 
 import (
 	"os"
-
-	"github.com/spf13/cobra"
 )
 
 func main() {
 	rootCmd := rootCommand()
 
-	rootCmd.RunE = func(cmd *cobra.Command, args []string) error {
```

Comment:
You can set this field on the `cmd` object that's created inside the `rootCommand` function instead, right below the `Long` description.